README.md

```diff
@@ -379,6 +379,12 @@ treated a value -- a list of which checkboxes are selected, for instance. When
 you want the user to be able to add items to the array, using `RepeatingForm`
 instead.
 
+### Text Arrays
+
+`converters.textStringArray`: raw value is a string with newlines. Value is an
+array of strings split by newline. Note that this is for use with strings with a newline
```

Comment:
The "note this is for use" is unnecessary now. Just write: You can use this with a textarea to edit an array of string split by newline.